packages/teleport-postprocessor-prettier-js/src/index.ts

```diff
@@ -15,12 +16,16 @@ export const createPostProcessor = (options: PostProcessorFactoryOptions = {}) =
   const fileType = options.fileType || FILE_TYPE.JS
   const formatOptions = { ...PRETTIER_CONFIG, ...options.formatOptions }
 
+  const jsParser = fileType === FILE_TYPE.TS ? parserTypescript : parserBabylon
+  const plugins = [jsParser, parserPostCSS]
+  const parser = fileType === FILE_TYPE.TS ? 'typescript' : 'babel'
```

Comment:
As I am trying a workaround here --> https://github.com/teleporthq/teleport-code-generators/pull/365/files#diff-cdb7f45628daba09fcfd879785f861e7R89 for adding reserved keywords.

Again prettier throws an error in the post-processing phase. Since prettier is currently using `babel-parser` for all TS files too. 

But this breaks when we try to do the same with more *typescript* specific syntaxes. So we need to use target-specific parsers to prettify the resultant code.

Reply:
Yeah, in a way that too makes sense. As we can just use either *prettier-ts* or *prettier-js* depending on the project. Will make those changes.